source/Logic.cpp

```diff
@@ -79,14 +80,17 @@ Logic::Logic(LevelScene &levelScene, Renderer &renderer, std::vector<AnimatedEnt
   , enemies(gameState.enemies, levelScene.enemies)
   , projectiles(gameState.projectiles, levelScene.projectiles)
   , entityFactory(renderer)
-  , pyEvaluate(gameState.players)
+  , pyEvaluate(gameState.players, gameState.enemies)
 {
   for (unsigned int i(0); i != 2; ++i) // TODO: obviously players should be passed as parameter or something.
     gameState.players.push_back(Player::makeArcher(Vect<2u, double>{(double)i, (double)i}));
   levelScene.setTerrain(gameState.terrain);
   enemies.add([this](){
       return entityFactory.spawnEnemy();
-    }, 100u, 0.5, Vect<2u, double>{7.5, 7.5});
+  }, 1u, 100u, 0.5, Vect<2u, double>{7.5, 7.5});
+  enemies.add([this](){
+      return entityFactory.spawnEnemy();
+  }, 2u, 100u, 0.5, Vect<2u, double>{3.0, 3.0});
```

Comment:
`1u` and `2u` aren't very descriptive. Declaring some constants might a good idea.